.github/workflows/codespell.yaml

```diff
@@ -0,0 +1,21 @@
+name: Codespell
+
+# Documentation:
+# https://help.github.com/en/articles/workflow-syntax-for-github-actions
+
+on: pull_request
+
+jobs:
+  codespell:
+    name: Codespell
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+      - name: Codespell
+        uses: codespell-project/actions-codespell@master
+        with:
+          skip: .git,./charts/kube-prometheus-stack/crds/crd-prometheuses.yaml,./charts/kube-prometheus-stack/crds/crd-alertmanagers.yaml,./charts/kube-prometheus-stack/templates/grafana/dashboards-1.14/apiserver.yaml
```

Comment:
Why are we skipping those? Are there spelling mistakes?

Reply:
Yes, there are. Those files are coming from upstream, so they shouldn't be modified here. I already fixed those typos upstream (prometheus-operator/prometheus-operator#3653), but I don't know how to pull those changes in here, as sync script is currently broken (#225).